### PR TITLE
Les-45-create number phone 

### PR DESCRIPTION
**Pull request by Kurilo-Pavel**

### CODE REVIEW

File: src/component/PhoneNumber.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {useState} from "react";
+import "./phoneNumber.scss"
+
+const PhoneNumber = () => {
+  const numberRegex = /[^\d]$/;
+  // const phoneNumberRegex = /[^\d]$/;
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:6]: Please remove commented code.

File: src/component/PhoneNumber.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const [value, setValue] = useState('');
+  const [isBlur, setIsBlur] = useState(false);
+
+  const fieldHandler = (e: React.FormEvent<HTMLInputElement>) => {
+    if (!numberRegex.test(e.currentTarget.value)) {
+      setValue(e.currentTarget.value);
+    }
+  }
+  const blurHandle = (e: React.FormEvent<HTMLInputElement>) => {
+      // if (phoneNumberRegex.test(e.currentTarget.value)) {
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:16]: Please remove commented code.

File: src/component/PhoneNumber.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div className={'phoneNumber'}>
+      <label className={'phoneNumberLabel'}>Phone number</label>
+      <input className={'phoneNumberInput'}
+             type={'text'}
+             onChange={fieldHandler}
+             onBlur={blurHandle}
+             value={value}
+             required/>
+      {/*{(isBlur) && <span className={'error'}>Please enter a phone number in international format</span>}*/}
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:29]: Please remove commented code.

File: src/component/PhoneNumber.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      setValue(e.currentTarget.value);
+    }
+  }
+  const blurHandle = (e: React.FormEvent<HTMLInputElement>) => {
+      // if (phoneNumberRegex.test(e.currentTarget.value)) {
+      //   setIsBlur(true);
+      // }
+  }
+  return (
+    <div className={'phoneNumber'}>
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:21]: className="phoneNumber"

File: src/component/PhoneNumber.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      <label className={'phoneNumberLabel'}>Phone number</label>
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:22]: class name in simple quotes:

className="phoneNumberLabel"
className="phoneNumberInput"

File: src/component/PhoneNumber.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      <input className={'phoneNumberInput'}
+             type={'text'}
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:24]: type="text"

File: src/component/PhoneNumber.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import {useState} from "react";
+import "./phoneNumber.scss"
+
+const PhoneNumber = () => {
+  const numberRegex = /[^\d]$/;
+  // const phoneNumberRegex = /[^\d]$/;
+  const [value, setValue] = useState('');
+  const [isBlur, setIsBlur] = useState(false);
+
+  const fieldHandler = (e: React.FormEvent<HTMLInputElement>) => {
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:10]: const handleFieldChange

File: src/component/PhoneNumber.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    if (!numberRegex.test(e.currentTarget.value)) {
+      setValue(e.currentTarget.value);
+    }
+  }
+  const blurHandle = (e: React.FormEvent<HTMLInputElement>) => {
```
Comment (Terapool) [inline src/component/PhoneNumber.tsx:15]: remove this handler as it is not used

File: src/component/phoneNumber.scss [hunk 9]
```diff
@@ -0,0 +1,30 @@
+$blue: #0747a6;
+
+.phoneNumber {
```
Comment (Terapool) [inline src/component/phoneNumber.scss:3]: .phoneNumber {
&nbsp;&nbsp;margin: 20px 0;

&nbsp;&nbsp;&Label {
&nbsp;&nbsp;...
&nbsp;&nbsp;}
&nbsp;&nbsp;&Input {
&nbsp;&nbsp;&nbsp;&nbsp;...
&nbsp;&nbsp;}

**Review by Terapool — CHANGES_REQUESTED** [review]

**Review by NatashaMikhaltsova — CHANGES_REQUESTED** [review]
Please rename folder with components to 'components'

**Review by OxygenO22 — APPROVED** [review]

File: src/component/phoneNumber.scss [hunk 10]
```diff
@@ -0,0 +1,30 @@
+$blue: #0747a6;
```
Comment (Terapool) [inline src/component/phoneNumber.scss:1]: Where do you use this $blue variable?

**Review by Terapool — CHANGES_REQUESTED** [review]

**Review by Terapool — APPROVED** [review]

**Review by NatashaMikhaltsova — APPROVED** [review]

**Review by DimitarShemet — APPROVED** [review]

**Pull request merged**